apps/client/src/features/rundown/event-block/__tests__/EventBlock.utils.test.ts

```diff
@@ -13,66 +13,15 @@ describe('formatDelay()', () => {
 
 describe('formatOverlap()', () => {
```

Comment:
Since these functions no longer contain logic, I am not sure how much we need to worry about the tests
either way, the test name should indicate what is under test

```suggestion
describe('formatGap()', () => {
```